src/shared/utils/getIteratorFn.js

```diff
@@ -29,7 +30,7 @@ var FAUX_ITERATOR_SYMBOL = '@@iterator'; // Before Symbol spec.
  * @param {?object} maybeIterable
  * @return {?function}
  */
-function getIteratorFn(maybeIterable) {
+function getIteratorFn(maybeIterable: any): ?(p: ReactElement<any, any, any>) => void {
```

Comment:
`any` is probably the right call here, could you put `?any` though. Even if it doesn't change anything in the type, it convey the fact that the function accepts a nullable value.